x-pack/legacy/plugins/triggers_actions_ui/np_ready/public/types.ts

```diff
@@ -5,6 +5,8 @@
  */
 import { capabilities } from 'ui/capabilities';
 import { TypeRegistry } from './application/type_registry';
+import { SanitizedAlert as Alert } from '../../../alerting/common';
+export { SanitizedAlert as Alert, AlertAction } from '../../../alerting/common';
```

Comment:
This allows us to use a SanitizedAlert in the UI but keep the typing clean by referring to it as the Alert.

This *could* be confusing if you move between the Backend plugin and the UI a lot, but I think once people get that the _internal Alert_ is different than the _external_ one, then keeping them both referred to as _Alert_ is actually simpler. No?